src/Stripe.net/Entities/Invoices/InvoiceTaxAmount.cs

```diff
@@ -2,19 +2,31 @@ namespace Stripe
 {
     using Newtonsoft.Json;
     using Stripe.Infrastructure;
+#if NET6_0_OR_GREATER
+    using STJS = System.Text.Json.Serialization;
+#endif
 
+#if NET6_0_OR_GREATER
+    [STJS.JsonConverter(typeof(STJMemberSerializationOptIn))]
```

Comment:
Have you considered a custom converter that you put onto the public expandable field?

Reply:
I did, but I couldn't get this work within a custom converter that is placed on the Id or the object.  When `Write` is called for a property, the converter excepts a writer that's already positioned for that field; to handle multiple fields I had to lift the converter up to a common level, which would be the class itself.  There are likely other options that involve restructuring how we represent expandable fields but they would have been too drastic a change for this PR, I think.